md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -277,6 +277,19 @@ please follow this procedure:
 * once done, you can restart your updated Bonita platform
 
 
+<a id="oracle12"/>
+
+## Migrating to Bonita 7.9+ using Oracle
+
+Bonita 7.9+ supports Oracle 12c version. To migrate to Bonita 7.9+ when using Oracle,
+please follow this procedure:
+* ensure your Bonita platform is shut down
+* run Bonita migration tool to update Bonita platform to version 7.8.4, following the procedure above
+* upgrade your Oracle database server to the version 12c
```

Comment:
Oracle 12c can mean 12.1 and 12.2. You can never say 12c without specifying (12.2.x.y), otherwise it doesn’t mean anything.

Reply:
As this PR is already closed, can you open a new suggestion through a PR, so that we can integrate it easily?
Thx

Reply:
That's what I thought, I wish I could and have given my input before it got closed. I'll do the PR myself, that will be easier ;)